chromadb/api/__init__.py

```diff
@@ -281,6 +281,22 @@ def _count(self, collection_id: UUID) -> int:
         """
         pass
 
+    @abstractmethod
+    def _dimensions(self, collection_id: UUID) -> int:
```

Comment:
A concern is this API implicltly prevents us / make maintaining a future multivector world harder